python_modules/libraries/dagster-dbt/dagster_dbt/asset_utils.py

```diff
@@ -609,6 +611,10 @@ def default_asset_check_fn(
         asset=asset_key,
         description=test_resource_props.get("meta", {}).get("description"),
         additional_deps=additional_deps,
+        metadata={
+            DAGSTER_DBT_TRANSLATOR_METADATA_KEY: dagster_dbt_translator,
+            DAGSTER_DBT_MANIFEST_METADATA_KEY: DbtManifestWrapper(manifest=manifest),
+        },
```

Comment:
Can you confirm that this metadata is hidden in the asset check UI?

Reply:
Working on this led me to notice that we don't actually show asset check definition metadata in the UI yet. A little more info in the description here: https://github.com/dagster-io/dagster/pull/23190.